docs/index.rst

```diff
@@ -27,15 +27,13 @@ governance and planning.
 
    team/index
    index-team_guides
-   index-team_governance
    contribute/index
 
 Why have a Team Compass?
 ========================
 
-This repository helps the JupyterHub team and Binder team set a weekly
-course for project activity. Our overriding goal is continuous team and
-project improvement.
+This repository helps the JupyterHub team and Binder team align on our structure, policy, and goals.
+Our overriding goal is continuous team and project improvement.
```

Comment:
I feel a bit nitpicky about "overriding" here. Maybe "overarching" is a better word?

I feel "overriding" is an opinionated action, like when you override a default value in code, it's because you know you want to do something different and discard whatever the software thinks you might want to do. Whereas "overarching" makes me think of many different actions that can contribute to the ultimate goal. Am I making sense?